src/parquet/util/rle-encoding.h

```diff
@@ -26,6 +26,7 @@
 #include "parquet/util/compiler-util.h"
```

Comment:
Since this is thirdparty code benchmarking / perf enhancements would be best tackled in a separate patch -- can you open a JIRA about this?